scripts/make_onnx_model.py

```diff
@@ -0,0 +1,61 @@
+
+# Usage: python3 script/make_onnx_model.py MODEL_PATH
+
+import sys
+import yaml
+import torch
+
+sys.path.append('./')
+
+from handyrl.environment import make_env
+from handyrl.model import to_torch, ModelWrapper
+from handyrl.util import map_r
+
+
+model_path = sys.argv[-1]
+saved_model_path = model_path + '.onnx'
+
+with open('config.yaml') as f:
+    args = yaml.safe_load(f)
+
+env = make_env(args['env_args'])
+model = env.net()
+model.load_state_dict(torch.load(model_path), strict=False)
+model.eval()
+model = ModelWrapper(model)
+
+model.load_state_dict(torch.load(model_path), strict=False)
+model.eval()
```

Comment:
Is model loading needed twice?

Reply:
Thanks! removed

Reply:
stopped importing and using ModelWrapper since it is used only for init_hidden()